screens/Posts.jsx

```diff
@@ -111,15 +111,17 @@ const Posts = ({ navigation }) => {
 
   // Create post
   const onPostSubmitClick = () => {
-    createPost({
-      username,
-      post: message.text,
-      businessMentions: findMentions(),
-    }).then(() => {
-      setMessage(null);
-      setClearInput(true);
-      refreshPosts();
-    });
+    if (message?.text) {
```

Comment:
is the ? supposed to be there?

Reply:
yeah otherwise if message is empty it'll throw error